app/main.js

```diff
@@ -0,0 +1,16 @@
+const { test } = require("./test");
+const { MajorColorNames, MinorColorNames } = require("./color-coder");
+
+function printColorCodes() {
```

Comment:
printColorCodes() could be moved to color-Coder.js or new file as this is a required feature which is production code.